main_app/models.py

```diff
@@ -27,4 +27,16 @@ def __str__(self):
         return self.name
     
     def get_absolute_url(self):
-        return reverse('detail', kwargs={'item_id': self.id})
\ No newline at end of file
+        return reverse('detail', kwargs={'item_id': self.id})
+    
+class HistoryLog(models.Model):
```

Comment:
Followed ERD as much as I could here. I have a feeling that we may need to revisit the fields for historylog because there may be a need for it to implement the parts of the function that aren't working just yet.

Reply:
Looks right to me based on the ERD, good job!